tests/mypy_test.py

```diff
@@ -414,10 +414,6 @@ def test_third_party_stubs(code: int, args: TestConfig) -> TestResults:
     files_checked = 0
 
     for distribution in sorted(os.listdir("stubs")):
-        if distribution in {"SQLAlchemy", "Flask-SQLAlchemy"}:
```

Comment:
These still crash locally for me if I run `mypy_test.py` twice in quick succession. It's an issue with the mypy cache; it should be fixed in mypy 0.970 when that comes out.

Reply:
Ah yes, I didn't try that. Same here.